app/services/analysis/gwas/map_csv_template_generator.rb

```diff
@@ -0,0 +1,15 @@
+class Analysis
+  class Gwas
+    class MapCsvTemplateGenerator
```

Comment:
Making these generators was not my brightest idea, static files would do. I did not plan to parametrize them. It actually might be useful but seems to be overkill (e.g. after uploading the genotype CSV the map template would be structured accordingly).